modules/n1ql/examples/transactions/multiple.n1ql

```diff
@@ -1,7 +1,7 @@
 /* tag::settings[] */
-\SET -txtimeout "2m"; -- <.>
-\SET -scan_consistency "not_bounded"; -- <.>
-\SET -durability_level "none"; -- <.>
+\SET -txtimeout "2m"; 
+\SET -scan_consistency "not_bounded";
+\SET -durability_level "none";
```

Comment:
This example file is used in multiple locations and _some_ of those need the callout numbers. Please reinstate. If you need to include an example file and exclude the callouts, use the `no-callouts` role, e.g. `[source.no-callouts,sqlpp]`

```suggestion
\SET -txtimeout "2m"; -- <.>
\SET -scan_consistency "not_bounded"; -- <.>
\SET -durability_level "none"; -- <.>
```